ee/tables/execparsers/nixenv/parser.go

```diff
@@ -0,0 +1,28 @@
+//go:build darwin
+// +build darwin
+
+package nixenv
+
+import (
+	"io"
+
+	"github.com/clbanning/mxj"
+)
+
+// parseNixXml implements the parser interface for nix-env XML data
+func (p *parser) parseNixXml(reader io.Reader) (any, error) {
+	// Read all data from the reader
+	rawdata, err := io.ReadAll(reader)
```

Comment:
Shouldn't need to pull it all into ram -- mxj has a reader interface. `NewMapXmlReader`